hack/csi_client/pkg/csi/client.go

```diff
@@ -14,21 +14,12 @@ import (
 )
 
 const (
-	attrIP           = "ip"
-	attrVolume       = "volume"
-	smbUser          = "smbUser"
-	smbPassword      = "smbPassword"
 	reqNodePublish   = "nodepublish"
 	reqNodeUnpublish = "nodeunpublish"
 )
 
 var (
-	goOs          = runtime.GOOS
-	errHost       = fmt.Errorf("host name is required")
-	errVolume     = fmt.Errorf("volume name is required")
-	errTargetPath = fmt.Errorf("targetPath is required")
-	errNoUsername = fmt.Errorf("username is required")
-	errNoPassword = fmt.Errorf("password is required")
+	goOs = runtime.GOOS
```

Comment:
do we use this anywhere?

Reply:
It was stubbed out for testing but I don't think there's OS-specific code anymore. I'll review it.